src/ds/test/json_schema.cpp

```diff
@@ -179,10 +179,25 @@ TEST_CASE("schema types")
   REQUIRE(schema["items"]["type"] == "array");
   REQUIRE(schema["items"]["items"].is_array());
   REQUIRE(schema["items"]["items"].size() == 2);
-  REQUIRE(schema["items"]["items"][0]["type"] == "number");
+  REQUIRE(schema["items"]["items"][0]["type"] == "integer");
   REQUIRE(schema["items"]["items"][1]["type"] == "string");
 }
 
+TEST_CASE("schema types II")
+{
+  std::map<size_t, float> m;
```

Comment:
Minor: Would be good to test double support as well? I may have syntax wrong, but I believe this will work:

```suggestion
TEST_CASE_TEMPLATE("schema types II", T, float, double)
{
  std::map<size_t, T> m;
```

Reply:
Good point, and the syntax is right. I've tweaked the integer test as well while I was at it.